packages/vscode-ui5-language-assistant/src/extension.ts

```diff
@@ -51,6 +56,34 @@ export async function activate(context: ExtensionContext): Promise<void> {
     clientOptions
   );
 
+  const jsonSchemaConfig:
```

Comment:
The type `const jsonSchemaConfig:
    | WorkspaceConfiguration
    | undefine` is strange.`

The API of .get()` is 
> get<T>(section: string, defaultValue: T): T

And the (expected) configuration is an Array of { fileMatch, url}

I think we should call the get with the relevant generic type:
- https://github.com/microsoft/vscode/blob/0c25f5a737a0c4b0945783588c13cfee11d0ecb1/src/vs/workbench/api/common/jsonValidationExtensionPoint.ts#L12-L15

Maybe this type is exported somewhere by \@vscode/types?